velox/functions/prestosql/aggregates/ApproxDistinctAggregate.cpp

```diff
@@ -504,7 +505,7 @@ exec::AggregateRegistrationResult registerApproxDistinct(
           const TypePtr& resultType,
           const core::QueryConfig& /*config*/)
           -> std::unique_ptr<exec::Aggregate> {
-        TypePtr type = argTypes[0]->isVarbinary() ? BIGINT() : argTypes[0];
+        TypePtr type = hllAsRawInput ? BIGINT() : argTypes[0];
```

Comment:
I'm not sure I understand this logic.

It looks like we only use 'type->kind()' and it is only used when hllAsRawInput is false. Hence, it seems we can simply use argTypes[0] for type.

```
const TypePtr& type = argTypes[0];
```